src/util/safelywritablefile.h

```diff
@@ -22,7 +22,16 @@ namespace mixxx {
 
 class SafelyWritableFile final {
   public:
-    SafelyWritableFile(QString origFileName, bool useTemporaryFile);
+    enum SafetyMode {
```

Comment:
```suggestion
    enum class SafetyMode {
```

I did not know that the enum prefix even works without "class"
We use CamelCase enum names for new enum classes